apps/website/src/components/SectionBlock.tsx

```diff
@@ -0,0 +1,67 @@
+"use client"
```

Comment:
Can `"use client"` be removed from here?

Reply:
Yes, it seems that it can be removed. When I created the component, I got an error that was fixed adding `"use client"` but I don't see the error anymore.